### PR TITLE
Sabsedev

### PR DESCRIPTION
**Pull request by chappy2**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/juanmi108/Super-Global-Hyper-Mega-Net-nosql/pull/36" target="_blank">ReviewNinja

**Comment by juanmi108** [comment]
could you please resolve the conflicts ?

**Comment by chappy2** [comment]
i resolve the conflicts when i merge

**Comment by juanmi108** [comment]
:+1:

File: public/lists/listsController.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+angular.module('app.lists', ['ngRoute', 'ngResource','app.listsService','app.todosService'])
+ .controller('ListController',['$scope','Lists','ListsPrivate','ListsPublic','ListsEdit','Todos','TodosEdit',function($scope,Lists,ListsPrivate,ListsPublic,ListsEdit,Todos,TodosEdit,user,index){
```
Comment (juanmi108) [inline public/lists/listsController.js]: where does the 'Lists' dependency comes from? it is used on line 32 to create a new list object.

**Pull request merged**